src/ast/definitions.ts

```diff
@@ -241,15 +241,12 @@ function* lookupInContractDefinition(
                     continue;
                 }
 
-                /**
-                 * Its a safe to assume V2 as its backward-compatible and we only use it internally here
-                 */
                 const sigHash =
                     child instanceof FunctionDefinition ||
                     child instanceof EventDefinition ||
                     (child instanceof VariableDeclaration &&
                         child.visibility === StateVariableVisibility.Public)
-                        ? inference.signatureHash(child, ABIEncoderVersion.V2)
+                        ? inference.signatureHash(child)
```

Comment:
This causes slight regression in contract archive:
```
Error: Getters of struct return type are not supported by ABI encoder v1
```
Apprx 13 samples are failing.